lib/FreeRTOS-Plus-POSIX/source/FreeRTOS_POSIX_clock.c

```diff
@@ -104,14 +101,6 @@ int clock_gettime( clockid_t clock_id,
     /* Silence warnings about unused parameters. */
     ( void ) clock_id;
 
-    /* Check tp. */
-    if( tp == NULL )
```

Comment:
If this check is removed, iStatus will never change,  so

>  if(iStatus ==0)

 may not be needed.

Reply:
updated in the second commit.